pkgs/development/embedded/arduino/arduino-language-server/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub }:
+
+{
+  pkg = buildGoModule rec {
+    pname = "arduino-language-server";
+    version = "0.7.1";
+
+    src = fetchFromGitHub {
+      owner = "arduino";
+      repo = "arduino-language-server";
+      rev = version;
+      hash = "sha256-FnkXtjktj22LCkmFPxzQ72qiVBC1WR/ZiwMsmZPL578=";
+    };
+
+    subPackages = [ "." ];
+
+    vendorHash = "sha256-Xa26ilo95sQ/6dGvl4gB2bb0vzWXr+WPKLezAnZPeqM=";
+
+    doCheck = false;
+
+    ldflags = [
+      "-s"
+      "-w"
+      "-X github.com/arduino/arduino-language-server/version.versionString=${version}"
+      "-X github.com/arduino/arduino-language-server/version.commit=unknown"
+    ] ++ lib.optionals stdenv.isLinux [ "-extldflags '-static'" ];
+
+    meta = with lib; {
+      inherit (src.meta) homepage;
+      description = "Arduino from the command line";
```

This is not an accurate description of a language server.

Indeed the page provides a better description.

```suggestion
      description = "An Arduino Language Server based on Clangd to Arduino code autocompletion";
```

---

Oops. I reused the expression from the cli. Sorry for the oversight. I'll make sure the rest of the metadata makes sense as well.